App.jsx

```diff
@@ -0,0 +1,11 @@
+import { useState } from 'react'
+
+function App() {
```

Comment:
Also, convert this into an arrow function, we will be using arrow function across the app.

Reply:
done... Kindly review PR